dist/setup-appliance.sh

```diff
@@ -709,9 +710,23 @@ fi
 
 
 if [[ ! $BOOTSTRAP_TEST_MODE == 1 && $0 != "-bash" ]];then
-
+  logline "Starting "`basename $0`" at "`date`
   NON_INTERACTIVE=0
 
+  if [ -f $PID_FILE ];then
+    APID=`cat $PID_FILE`
+    if [ ! -f /proc/$APID/status ];then
```

Comment:
If you switch the conditions around it's way easier to understand IMHO

Reply:
I didn't get the point - please explain!

Reply:
```bash
if true; then
...
else
...
fi
```

vs.

```bash
if !true; then
...
else
...
fi
```

Reply:
Feel free to send an PR - and test it - if you really unhappy with the current solution.
Testing these changes is a lot of effort and I'm not willing to do this effort only for "coding style"

Reply:
I'm with Henne here. I will create a PR later, with his suggestion, so we can also merge this PR.